trg_ros2_node:
  ros__parameters:
    ros2:
      isDebug:                    true
      frameId:                    "map"
      publishRate:                10.0
      debugRate:                  1.0
      topic:
        input:
          egoPose:                "/trg/input/default_pose"
          egoOdom:                "/ground_truth_pose"
          obsCloud:               "/trip/trip_updated/alocal_cloud"
          goal:                   "/move_base_simple/goal"
        output:
          preMap:                 "/trg/output/default_preMap"
          goal:                   "/trg/output/default_goal"
          path:                   "/trg/output/default_path"
        debug:
          globalTRG:           "/trg/debug/default_globalTRG"
          localTRG:            "/trg/debug/default_localTRG"
          obsMap:              "/trg/debug/default_obsMap"
          pathInfo:              "/trg/debug/default_pathInfo"


      timer:
        graphRate: 5.0
        planningRate: 5.0
        publishRate: 10.0
        debugRate: 1.0

      map:
        isPrebuiltMap: true
        preMapPath: "maps/sim_mountain_0.1.pcd"

      trg:
        isPrebuiltTRG: false
        preTrgPath: "trg/20240605_KI_exproom_terrain_trg.pcd"
        isUpdate: true
        expandDist: 0.6
        robotSize: 0.5
        sampleNum: 25
        heightThreshold: 0.15
        collisionThreshold: 0.1
        updateCollisionThreshold: 0.5
        safetyFactor: 3.0

      planner:
        goalTolerance: 0.8
